.changeset/olive-toys-try.md

```diff
@@ -0,0 +1,6 @@
+---
+'@react-select/docs': minor
+'react-select': minor
```

Comment:
This would be a major release, because a major update to emotion effectively changes our public API for react-select (even though it should be pretty painless for users to upgrade)

```suggestion
'react-select': major
```